src/refiners/EN/ENPrioritizeSpecificDateRefiner.js

```diff
@@ -0,0 +1,109 @@
+/*
+
+*/
+var ParsedComponents = require('../../result').ParsedComponents;
+var Refiner = require('../refiner').Refiner;
+
+
+var PATTERN = new RegExp("^\\s*(at|after|before|on|,|-|\\(|\\))?\\s*$");
+
+function isMoreSpecific(prevResult, currResult) {
+    var moreSpecific = false;
+
+    if (prevResult.start.isCertain('year'))
+        if (!currResult.start.isCertain('year'))
+            moreSpecific = true;
```

Comment:
Actually for "next week" I don't assign a year, since a change of year/month can occur in the next week. For example if today was 25 Dec 2017, next week could be 31 Dec 2017 or 1 Jan 2018.